corebehrt/common/config.py

```diff
@@ -1,25 +1,26 @@
 import importlib
 import yaml
-
+import json
+from os.path import join
 
 class Config(dict):
     """Config class that allows for dot notation."""
     def __init__(self, dictionary=None):
-        super().__init__()
+        super(Config, self).__init__()
```

Comment:
This is not needed, we can just do `super().__init__()`

And we probably need to have `super().__init__(dict)` and we can leave the below code out (?)

Reply:
Resolved by https://github.com/FGA-DIKU/EHR/pull/5/commits/1d62fbd7a800cf5782dec9438f4f14ccb3c80994